test/Libraries/SystemTestServices/SystemTestServices.csproj

```diff
@@ -8,24 +11,34 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>SystemTestServices</RootNamespace>
     <AssemblyName>SystemTestServices</AssemblyName>
+      <EnableDefaultPageItems>false</EnableDefaultPageItems>
```

Comment:
does this project contain xaml? I thought this was only required for controls?

Reply:
I think you're right, will remove.